queries.sql

```diff
@@ -7,4 +7,75 @@ SELECT date_of_birth from animals WHERE name  IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT name from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* Start transaction query for species column */
+
+BEGIN;
+  UPDATE animals SET species = 'unspecified';
+
+  select * from animals;
+
+ROLLBACK TRANSACTION;
+
```

Comment:
- [ ] It is a good practice to always verify your changes in a transaction. Please verify that the change did rollback by adding: `SELECT species from animals;` after the rollback statement.